packages/api/src/codegen/factory.ts

```diff
@@ -4,9 +4,14 @@ import type Oas from 'oas';
 
 import TSGenerator from './languages/typescript/index.js';
 
-export enum SupportedLanguages {
-  JS = 'js',
-}
+export const SupportedLanguages = {
```

Comment:
we can't use TS enums here anymore since we're deriving our supported languages from the JSON schema